SMLHelper/Patchers/EnumPatching/EntityCellPatcher.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+namespace SMLHelper.V2.Patchers.EnumPatching
+{
+    internal class EntityCellPatcher
+    {
+        [HarmonyPatch(typeof(EntityCell),nameof(EntityCell.SerializeWaiterDataAsync))]
+        internal static class EntityCell_SWDA_Patch
+        {
+            [HarmonyTranspiler]
+            internal static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> list_)
+            {
+                var list = new List<CodeInstruction>(list_);
+                for(var i = 0;i < list.Count;i++)
+                {
+                    if (i == 51)
```

Comment:
Is there any better way to identify which instruction to replace beyond counting?  
If this method ever updates, this patch will break.